txn_writer.go

```diff
@@ -418,38 +418,51 @@ fail:
 }
 
 // PrintPreparedActions logs the actions a call to Write would perform.
-func (sw *SafeWriter) PrintPreparedActions(output *log.Logger) error {
+func (sw *SafeWriter) PrintPreparedActions(verbose bool, output *log.Logger) error {
 	if sw.HasManifest() {
-		output.Printf("Would have written the following %s:\n", ManifestName)
-		m, err := sw.Manifest.MarshalTOML()
-		if err != nil {
-			return errors.Wrap(err, "ensure DryRun cannot serialize manifest")
+		if verbose {
+			m, err := sw.Manifest.MarshalTOML()
+			if err != nil {
+				return errors.Wrap(err, "ensure DryRun cannot serialize manifest")
+			}
+			output.Printf("Would have written the following %s:\n%s\n", ManifestName, string(m))
+		} else {
+			output.Printf("Would have written %s.\n", ManifestName)
 		}
-		output.Println(string(m))
 	}
 
 	if sw.writeLock {
 		if sw.lockDiff == nil {
-			output.Printf("Would have written the following %s:\n", LockName)
-			l, err := sw.lock.MarshalTOML()
-			if err != nil {
-				return errors.Wrap(err, "ensure DryRun cannot serialize lock")
+			if verbose {
+				l, err := sw.lock.MarshalTOML()
+				if err != nil {
+					return errors.Wrap(err, "ensure DryRun cannot serialize lock")
+				}
+				output.Printf("Would have written the following %s:\n%s\n", LockName, string(l))
+			} else {
+				output.Printf("Would have written %s.\n", LockName)
 			}
-			output.Println(string(l))
 		} else {
-			output.Printf("Would have written the following changes to %s:\n", LockName)
-			diff, err := formatLockDiff(*sw.lockDiff)
```

Comment:
The code is no longer printing the lock diff when `-v` is not specified which seems a little overly terse. I prefer that it is always printed during a dry-run, not just on verbose.

Otherwise I either have to settle for no useful output (other than knowing that dep found changes), or I have to wade through dozens of lines of solve output just to see what new package would be written to my lock.

e.g. Terse

```
$ dep ensure -dry-run
Would have written changes to Gopkg.lock.
Would have written 16 projects to the vendor directory.
```

Verbose 
```
$ dep ensure -dry-run -v
Root project is "github.com/meyskens/readmyage-api"
 3 transitively valid internal packages
 6 external packages imported from 5 projects
(0)   ✓ select (root)
(1)	? attempt github.com/labstack/echo with 2 pkgs; at least 1 versions to try
(1)	    try github.com/labstack/echo@v3.2.1
(1)	✓ select github.com/labstack/echo@v3.2.1 w/2 pkgs
(2)	? attempt github.com/dgrijalva/jwt-go with 1 pkgs; at least 1 versions to try
(2)	    try github.com/dgrijalva/jwt-go@v3.0.0
(2)	✓ select github.com/dgrijalva/jwt-go@v3.0.0 w/1 pkgs
(3)	? attempt github.com/stretchr/testify with 1 pkgs; at least 1 versions to try
(3)	    try github.com/stretchr/testify@v1.1.4
(3)	✓ select github.com/stretchr/testify@v1.1.4 w/1 pkgs
(4)	? attempt github.com/davecgh/go-spew with 1 pkgs; at least 1 versions to try
(4)	    try github.com/davecgh/go-spew@v1.1.0
(4)	✓ select github.com/davecgh/go-spew@v1.1.0 w/1 pkgs
(5)	? attempt golang.org/x/crypto with 1 pkgs; at least 1 versions to try
(5)	    try golang.org/x/crypto@master
(5)	✓ select golang.org/x/crypto@master w/2 pkgs
(6)	? attempt github.com/labstack/gommon with 4 pkgs; at least 1 versions to try
(6)	    try github.com/labstack/gommon@0.2.2
(6)	✓ select github.com/labstack/gommon@0.2.2 w/4 pkgs
(7)	? attempt github.com/mattn/go-colorable with 1 pkgs; at least 1 versions to try
(7)	    try github.com/mattn/go-colorable@v0.0.9
(7)	✓ select github.com/mattn/go-colorable@v0.0.9 w/1 pkgs
(8)	? attempt gopkg.in/resty.v0 with 1 pkgs; at least 1 versions to try
(8)	    try gopkg.in/resty.v0@v0.13
(8)	✓ select gopkg.in/resty.v0@v0.13 w/1 pkgs
(9)	? attempt github.com/mattn/go-isatty with 1 pkgs; at least 1 versions to try
(9)	    try github.com/mattn/go-isatty@v0.0.2
(9)	✓ select github.com/mattn/go-isatty@v0.0.2 w/1 pkgs
(10)  ? attempt github.com/pmezard/go-difflib with 1 pkgs; at least 1 versions to try
(10)      try github.com/pmezard/go-difflib@v1.0.0
(10)  ✓ select github.com/pmezard/go-difflib@v1.0.0 w/1 pkgs
(11)  ? attempt github.com/valyala/fasttemplate with 1 pkgs; at least 1 versions to try
(11)      try github.com/valyala/fasttemplate@master
(11)  ✓ select github.com/valyala/fasttemplate@master w/1 pkgs
(12)  ? attempt github.com/valyala/bytebufferpool with 1 pkgs; at least 1 versions to try
(12)      try github.com/valyala/bytebufferpool@master
(12)  ✓ select github.com/valyala/bytebufferpool@master w/1 pkgs
(13)  ? attempt github.com/sdboyer/deptest with 1 pkgs; 4 versions to try
(13)      try github.com/sdboyer/deptest@v1.0.0
(13)  ✓ select github.com/sdboyer/deptest@v1.0.0 w/1 pkgs
(14)  ? attempt golang.org/x/sys with 1 pkgs; at least 1 versions to try
(14)      try golang.org/x/sys@master
(14)  ✓ select golang.org/x/sys@master w/1 pkgs
(15)  ? attempt golang.org/x/net with 1 pkgs; at least 1 versions to try
(15)      try golang.org/x/net@master
(15)  ✓ select golang.org/x/net@master w/2 pkgs
(16)  ? attempt golang.org/x/text with 2 pkgs; at least 1 versions to try
(16)      try golang.org/x/text@master
(16)  ✓ select golang.org/x/text@master w/9 pkgs
  ✓ found solution with 30 packages from 16 projects

Solver wall times by segment:
     b-source-exists: 3.943060411s
         b-list-pkgs: 3.493432407s
              b-gmal: 3.359844321s
  b-deduce-proj-root:  97.322674ms
             satisfy:   4.855499ms
         select-atom:   4.218009ms
            new-atom:   1.096136ms
         select-root:    151.083µs
               other:     74.229µs
     b-list-versions:     26.213µs

  TOTAL: 10.904080982s

Would have written the following changes to Gopkg.lock:
Memo: 4de59b7a3096f65ac52284a5a5e07c3ee048c47d67893392d1c8f2f409cfc98f -> 229dbcd910e23d4cc7f9713836f3243a12c5029b594ad68bc4dc78623ec26349

Add:
[[projects]]
  name = "github.com/sdboyer/deptest"
  packages = ["."]
  revision = "ff2948a2ac8f538c4ecd55962e919d1e13e74baf"
  version = "v1.0.0"


Would have written the following 16 projects to the vendor directory:
github.com/labstack/echo@v3.2.1 with packages: [. middleware]
github.com/dgrijalva/jwt-go@v3.0.0 with packages: [.]
github.com/davecgh/go-spew@v1.1.0 with packages: [spew]
github.com/mattn/go-colorable@v0.0.9 with packages: [.]
github.com/mattn/go-isatty@v0.0.2 with packages: [.]
github.com/pmezard/go-difflib@v1.0.0 with packages: [difflib]
github.com/valyala/fasttemplate@master with packages: [.]
github.com/stretchr/testify@v1.1.4 with packages: [assert]
golang.org/x/crypto@master with packages: [acme acme/autocert]
github.com/labstack/gommon@0.2.2 with packages: [bytes color log random]
github.com/sdboyer/deptest@v1.0.0 with packages: [.]
golang.org/x/sys@master with packages: [unix]
gopkg.in/resty.v0@v0.13 with packages: [.]
github.com/valyala/bytebufferpool@master with packages: [.]
golang.org/x/net@master with packages: [idna publicsuffix]
golang.org/x/text@master with packages: [internal/gen internal/triegen internal/ucd secure/bidirule transform unicode/bidi unicode/cldr unicode/norm unicode/rangetable]
```